plugins/lcvfeatures2d/src/qdescriptorextractor.cpp

```diff
@@ -46,19 +52,26 @@ QDescriptorExtractor::QDescriptorExtractor(cv::Ptr<cv::DescriptorExtractor> extr
 QDescriptorExtractor::~QDescriptorExtractor(){
     delete m_descriptors;
 }
-
+/**
+ * \brief Initialize QDescriptorExtractor
+ */
 void QDescriptorExtractor::initialize(const QVariantMap &){}
-
+/**
+ * \brief Creates a descriptor extractor by name.
+ */
 void QDescriptorExtractor::initializeExtractor(cv::Ptr<cv::DescriptorExtractor> extractor){
     m_extractor = extractor;
     compute();
 }
-
+/// \todo
```

Comment:
Implementation from QQuickItem. Triggers a calculation of descriptors if the keypoints have been set.